controller/src/main/java/io/pravega/controller/fault/ControllerClusterListener.java

```diff
@@ -57,7 +58,7 @@ public ControllerClusterListener(final Host host, final Cluster cluster,
         this.host = host;
         this.cluster = cluster;
         this.executor = executor;
-        this.sweepers = sweepers;
+        this.sweepers = Lists.newArrayList(sweepers);
```

Comment:
no, this list is internal to this class, so making it immutable is not required because we have control over what happens to this list. 
the original comment tom had was valid because we were setting reference to an externally supplied mutable list which the caller can himself modify.